client/app/telecom/pack/xdsl/access/comfortExchange/pack-xdsl-access-comfort-exchange.html

```diff
@@ -0,0 +1,60 @@
+<div class="telecom-pack-xdsl-access-comfort-exchange">
+
+    <div class="widget-presentation">
+        <h2 class="widget-presentation-title" data-translate="xdsl_access_comfort_exchange_title">
+        </h2>
+
+        <tuc-toast-message></tuc-toast-message>
+
+        <div data-ng-if="!$ctrl.exchange.isSucceed"
+             data-translate="xdsl_access_comfort_exchange_waiting"></div>
+
+        <div data-ng-if="$ctrl.exchange.isSucceed">
+            <h3 data-translate="xdsl_access_comfort_exchange_bc_title"></h3>
+            <div class="form-group row">
+                <label class="col-sm-2 col-form-label col-form-label-sm"
+                       data-translate="xdsl_access_comfort_exchange_bc_number"></label>
+                <span class="oui-tile__description" data-ng-bind="$ctrl.exchange.order.id"></span>
+            </div>
+            <div class="form-group row">
+                <label class="col-sm-2 col-form-label col-form-label-sm"
+                       data-translate="xdsl_access_comfort_exchange_bc_date"></label>
+                <span class="oui-tile__description" data-ng-bind="($ctrl.exchange.order.date | date: 'dd/MM/yyyy HH:mm:ss' )"></span>
+            </div>
+            <div class="form-group row">
+                <label class="col-sm-2 col-form-label col-form-label-sm"
+                       data-translate="xdsl_access_comfort_exchange_price_without_tax"></label>
+                <span class="oui-tile__description" data-ng-bind="$ctrl.exchange.order.priceWithoutTax"></span>
+            </div>
+            <div class="form-group row">
+                <label class="col-sm-2 col-form-label col-form-label-sm"
+                       data-translate="xdsl_access_comfort_exchange_tax"></label>
+                <span class="oui-tile__description" data-ng-bind="$ctrl.exchange.order.tax"></span>
+            </div>
+            <div class="form-group row">
+                <label class="col-sm-2 col-form-label col-form-label-sm"
+                       data-translate="xdsl_access_comfort_exchange_price_with_tax"></label>
+                <span class="oui-tile__description" data-ng-bind="$ctrl.exchange.order.priceWithTax"></span>
+            </div>
+
+            <p>
+                <a target="_blank"
+                   class="btn btn-primary"
+                   rel="noopener"
+                   data-ng-href="{{:: $ctrl.exchange.order.url }}">
+                    <span data-translate="xdsl_access_comfort_exchange_bc_link"></span>
+                    <i class="fa fa-external-link"
```

Comment:
```suggestion
                    <span class="fa fa-external-link"
```

`i` is supposed to be used to describe unusual text (text in another language for example), `span` has no meaning